rosbag2_storage/src/rosbag2_storage/base_io_interface.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
doesn't really matter - but fyi when we introduce a new source file you can put the Amazon header in the copyright

Reply:
Fixed